CSS/Components/button.css

```diff
@@ -0,0 +1,84 @@
+.btn {
+  padding: 0.4rem 0.6rem;
+  border-radius: 4px;
+  font-size: 1.2rem;
+  font-weight: 500;
+  margin: 0 10px;
+  border: none;
+
```

Comment:
When you click the button on mobile phone, it will show an outline by default which is a bad UX. Use below code to remove it.
.btn: focus { outline:none}